py3status/formatter.py

```diff
@@ -345,6 +345,10 @@ def _check_valid_condition(self, get_params):
             variable = None
         value = self.value
 
+        # reject the condition?
```

Comment:
This comment does not help.  It spreads confusion.

There are many reasons for comments.  They are there to explain why we are doing things `reject the condition?` helps no-one.  Here we are dealing with a special case that you feel is a bug and worth of a PR to fix.  We should have an explanation of why we are doing this test.

But really it would help if we decide on our logic here. For example should `None=None`?

Reply:
I'll write a different comment.

For me...
`None=None` is rejected if it's a None. Users should use `if=None` instead.
`None=None` is accepted if it's a string 'None`.

I thought about a condition for `is`, but I'm not sure if that's worth supporting. I have a branch for `==`, `>=`, `<=`, etc... as I don't like doing `if=volume>9` to get 10 or above.